addons/mail/static/src/core/common/out_of_focus_service.js

```diff
@@ -120,10 +125,10 @@ export class OutOfFocusService {
      * @param {string} title
      * @param {string} message
      */
-    sendNativeNotification(title, message) {
+    sendNativeNotification(title, message, icon) {
         const notification = new Notification(title, {
             body: message,
-            icon: "/mail/static/src/img/odoobot_transparent.png",
+            icon: icon,
```

Comment:
Nitpick: no need to pass key, value separately as both has same name. It would help in converting the params into inline.